src/styles/index.css

```diff
@@ -0,0 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Serif+Pro:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&display=swap');
+@font-face {
+    font-family: "Stanford";
+    src: url("https://www-media.stanford.edu/assets/fonts/stanford.woff") format("woff"), url("https://www-media.stanford.edu/assets/fonts/stanford.ttf") format("truetype");
+    font-weight: 300;
+    font-display: swap;
+}
```

Comment:
Any reason you are including this here instead of importing https://github.com/SU-SWS/decanter/blob/v7/fonts.css

Reply:
Yes, I don't want to import the slab and monospace fonts that we most likely won't use in this project 😄